src/content/docs/build/index.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Build
+layout: "@layouts/DocsPageLayout.astro"
+---
+
+import BuildCloud from "@cmp/core/clouds/BuildCloud.svelte";
+
+<BuildCloud/>
+
+Welcome to Deta Space. You’re on an exciting journey building for a whole new cloud computer that's designed around you. We’re glad to have you here.
+
+Unlike other cloud computers you’ve built for, Space is also a personal computer. You can build apps for yourself. You can [use](/docs/en/use) and extend [3rd party apps](/docs/en/use/space-apps/discovery) that put you in the drivers seat.  
+
+You can also [share your creations with the world](/docs/en/publish). And when you do, others can run your apps in their own cloud. 
```

Comment:
Feel like running something on your cloud computer is easier to understand and fits better than running something in your own cloud.

```suggestion
You can also [share your creations with the world](/docs/en/publish). And when you do, others can run your apps on their own cloud computer. 
```